packages/create-node-app-generator/app/templates/express/_config.json

```diff
@@ -0,0 +1,3 @@
+{
+  "port": 3000
```

Comment:
Wouldn't the base config need connectSecret and client id? We should be instantiating auth somewhere right? https://foundations-documentation.reapit.cloud/app-development/connect-session#node-usage

Reply:
Will we always need this by default you think? Could use api-key